docker/Dockerfile

```diff
@@ -6,12 +6,15 @@ FROM "${base_image}"
 
 ARG python_version="3.7"
 ARG release_version="nightly"
+ARG xla_branch=""
+ARG example_branch="master"
 ARG cuda="0"
 ARG cuda_compute="3.7,7.0,7.5,8.0"
 ARG cxx_abi="1"
 ARG tpuvm=""
 ARG bazel_jobs=""
 ARG git_clone="true"
+ARG build_cpp_tests="0"
```

Comment:
hmm, we used to run all cpp tests before publihsing the wheel, I disabled that because of the MKL issue. If that's fixed, we should consider reenabling that check.

Reply:
It's fixed, and it's set to false as default. Now we can toggle it directly in the trigger definition :)